v-dev.sql

```diff
@@ -0,0 +1,16 @@
+-- OCD - 2351 - nonconformity chart statistics
+
+DROP TABLE openchpl.nonconformity_type_statistics;
```

Comment:
Should this be "DROP TABLE IF EXISTS openchpl....."?

Also, needs to be removed from the v14.something file